zipline/pipeline/expression.py

```diff
@@ -328,7 +326,9 @@ def __repr__(self):
 
     def short_repr(self):
         """Short repr to use when rendering Pipeline graphs."""
-        return "Expression: {expr}".format(
-            typename=type(self).__name__,
-            expr=self._expr,
+        final = re.sub(r"[-+]?\d*\.\d+",
+                       lambda x: format(float(x.group(0)), '.2E'),
+                       self._expr)
+        return "Expression:\l{}\l".format(
```

Comment:
In our other uses of `\l` it looks like we're putting two leading spaces after the line break. Any reason not to do that here too"

Reply:
Sure, I can add that in